packages/dependency-profiles/src/index.js

```diff
@@ -0,0 +1,314 @@
+// This file was generated by 'build.js'
+/* eslint-disable */
+module.exports = {
+  "latest": {
```

Comment:
Do we still support users on other versions of react-native?

Reply:
We'll start with RN `0.68` and "freeze" older profiles as we update RN versions since we don't currently maintain older releases.